go.mod

```diff
@@ -20,10 +20,10 @@ require (
 	github.com/jhump/protoreflect v1.8.2
 	github.com/json-iterator/go v1.1.12
 	github.com/tidwall/gjson v1.9.3
-	golang.org/x/net v0.0.0-20221014081412-f15817d10f9b
-	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
-	golang.org/x/sys v0.0.0-20220817070843-5a390386f1f2
-	golang.org/x/tools v0.1.12
+	golang.org/x/net v0.12.0
```

Comment:
These changes cause the compatibility tests to fail due to go version-related issues (ex. io/fs is supported after go1.16). You can revert these changes.